.travis.yml

```diff
@@ -12,26 +12,30 @@ install:
   - sudo pip install -r requirements-dev.txt
 jobs:
   include:
-    - stage: Tests
-      name: "Default (minikube with latest k8s)"
+    - name: "minishift"
+      env:
+        - CPLATFORM=minishift
+        - KUBEVIRT=v0.16.0
```

Comment:
v0.16.2 - latest released

Reply:
0.16.1 and 0.16.2 are tagged as pre–release, which is why I didn't use them. Dunno if those tags can be ignored or if they actually mean something.

Reply:
There is `v0.17.0` available please change.